Pocket.xcodeproj/project.pbxproj

```diff
@@ -696,6 +861,7 @@
 				CODE_SIGN_IDENTITY = "Apple Distribution: Read It Later, Inc (EX3VH4YFCH)";
 				CODE_SIGN_STYLE = Manual;
 				DEVELOPMENT_TEAM = EX3VH4YFCH;
+				"DEVELOPMENT_TEAM[sdk=iphoneos*]" = EX3VH4YFCH;
```

Comment:
Is adding the `sdk=iphoneos*` constraint truly necessary here? I feel like having it adds noise/complication to our configuration. Personally, I'd prefer to keep this out unless it's 100% necessary. (same goes for the configuration of the newly added targets)

Reply:
this is xcode autoadding stuff. Will verify that removing it doesn't break things and will update.